contribute/general/gokit.md

```diff
@@ -1,40 +1,40 @@
 
-## How to convert existing projects to have “kit”
-OpenEBS being a container native project is delivered via set of containers. For instance, with OpenEBS 0.3 release we have the following active maya related projects:
+## How to convert existing projects to have “kit”?
+OpenEBS being a container native project is delivered via a set of containers. For instance, with OpenEBS 0.3 release we have the following active Maya related projects:
 
-* openebs/maya aka maya-cli : is the command line interface like kubectl for interacting with maya services for performing storage operations.
-* openebs/mayaserver : or m-apiserver abstracts a generic volume api that can be used to provision OpenEBS Disks using containers launched using the CO like K8s, nomad etc.,
+* openebs/maya aka maya-cli : is the command line interface like kubectl for interacting with Maya services for performing storage operations.
+* openebs/mayaserver : or m-apiserver abstracts a generic volume API that can be used to provision OpenEBS Disks using containers launched using the CO like K8s, nomad etc.,
 * openebs/openebs-k8s-provisioner : is the K8s controller for dynamically creating OpenEBS PVs
 With these projects, we are already seeing how code gets duplicated when each of these projects are independently developed. For example maya-cli and openebs-k8s-provisioner both need to interact with maya-apiserver, which resulted in maya-apiserver-client code being written in maya-cli and openebs-k8s-provisioner. Similarly, openebs-k8s-provisioner and maya-apiserver have duplicated code w.r.t to accessing the K8s services.
 
-To avoid this duplicity of code using the kit project, we are transforming openebs/maya into a Kit project for the Application projects like maya-apiserver, openebs-k8s-provisioner and many more coming up in the future. openebs/maya contains all the kubernetes & nomad API’s, common utilities etc. needed for development of maya-apiserver and maya-storage-bot. In the near future, we are trying to push our custom libraries to maya. So that, it will become a promising Go kit project for OpenEBS community.
+To avoid this duplicity of code using the kit project, we are transforming openebs/maya into a Kit project for the Application projects like maya-apiserver, openebs-k8s-provisioner and many more coming up in the future. openebs/maya contains all the Kubernetes & nomad API’s, common utilities etc. needed for development of maya-apiserver and maya-storage-bot. In the near future, we are trying to push our custom libraries to Maya. So that, it will become a promising Go kit project for OpenEBS community.
```

Comment:
Application  = application
& = and
API's = APIs
Maya. So that,  = Maya, so that it